tests/Unit/Database/Schema/BlueprintTest.php

```diff
@@ -36,23 +37,24 @@ public function it_gets_the_index_alias(string $expected, $alias = null)
     /**
      * getAlias data provider.
      */
-    public function get_alias_data_provider(): array
+    public function get_alias_data_provider():array
     {
         return [
             'alias not provided' => ['indices_dev'],
-            'alias provided' => ['alias_dev', 'alias']
+            'alias provided'     => ['alias_dev', 'alias'],
         ];
     }
 
     /**
      * It gets the document type.
+     *
      * @test
-     * @covers \DesignMyNight\Elasticsearch\Database\Schema\Blueprint::getDocumentType
+     * @covers       \DesignMyNight\Elasticsearch\Database\Schema\Blueprint::getDocumentType
      * @dataProvider get_document_type_data_provider
      */
     public function it_gets_the_document_type(string $expected, $documentType = null)
     {
-        if (isset($documentType)) {
+        if(isset($documentType)) {
```

Comment:
Going to ignore because its a test file...and I am a generous God.

<img src="https://media1.tenor.com/images/c2041dddd5a1286480756fcb9ab4e222/tenor.gif?itemid=11629859"/>

Reply:
I'm not so kind to myself :joy: